### ISSUE TITLE
Education Component Accessibility Issue

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/dev-cv#34]
### Description

1. Make sure all input fields are accessible.

Currently, the axe DevTools are showing that

> 
> Issue Description
> 
> Ensures form field does not have multiple label elements

To reproduce the issue, install the axe DevTools extension and run it.

### Additional information

For further instructions on how to make the component accessible, please read the following:

- [Form field multiple labels](https://dequeuniversity.com/rules/axe/4.4/form-field-multiple-labels?application=AxeFirefox)
- [IDs used in ARIA and labels must be unique ](https://dequeuniversity.com/rules/axe/4.4/duplicate-id-aria?application=AxeFirefox)

**Comment by utkarsh1311** [comment]
I would like to work on it.

**Comment by YuriDevAT** [comment]
Sure @utkarsh1311, go ahead 🙂
